styles/buttons.scss

```diff
@@ -58,6 +58,28 @@
   }
 }
 
+// help icon button
+.btn-help {
+  @include layout;
```

Comment:
you can also `@include common` here and add the margin props to the component. Also extend the type with `CommonButtonInterface`. Check how the other components do it for guidance.

Reply:
`CommonButtonInterface` also supports `disabled`, so can you also add a `disabled` prop and make the opacity `var(--opacity-disabled)` with `&:disabled`.